.github/workflows/kcp-image.yaml

```diff
@@ -45,6 +45,17 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ github.token }}
 
+  deploy:
+    if: "github.repository_owner == 'kcp-dev' && github.ref_name == 'main'"
+    name: Deploy KCP
+    needs: build
+    environment: unstable
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get the short sha
+        id: vars
+        run: echo "::set-output name=sha_short::$(echo ${{ github.sha }} | cut -b -7)"
```

Comment:
Given this is on the `release-0.3` branch, we probably don't need it here?

Reply:
Line 49 disables this for all branches except main.  but we could just delete this whole job from this branch.  I can go either way.

Reply:
Let's delete it from the branch